config/install/user.role.authenticated.yml

```diff
@@ -29,6 +30,7 @@ permissions:
   - 'edit any document media'
   - 'edit any image media'
   - 'edit any remote_video media'
+  - 'edit domain path entity'
```

Comment:
I'm proposing we remove the authenticated user role config #168 

These permissions should be added to [microsites_trusted_editor](https://github.com/localgovdrupal/localgov_microsites_group/blob/1.x/config/install/user.role.microsites_trusted_editor.yml) and [microsites_controller](https://github.com/localgovdrupal/localgov_microsites_group/blob/1.x/config/install/user.role.microsites_controller.yml) as required.

Reply:
Which way round we do this?

Reply:
Just checking the permission. This is required to set a path on an entity. Is this only microsites_trusted_editor, or is it anyone who can post content?